apps/apps.humanatlas.io/src/app/pages/us1/cell-population-predictions/cell-population-predictions.component.scss

```diff
@@ -26,6 +26,8 @@ button {
 .table-container {
   margin-top: 3rem;
   margin-bottom: 0.3125rem;
+  max-height: 26rem;
+  overflow-y: auto;
```

Comment:
Only the table content should scroll. Not labels, headers, etc. Use ng-scrollbar. Check us2 tables for implementation details